source-reminder/lambda_function.py

```diff
@@ -51,13 +54,35 @@ def request_data(limit, start_date, end_date):
     return 0
 
 
+def request_author(slug):
+    try:
+        response_data = requests.get(API_ENDPOINT + slug)
+        return response_data
+    except HTTPError as errh:
+        print("HTTP Error:", errh)  # Should Slack send a message to the channel, stating that an error occurred?
+    except ConnectionError as errc:
+        print("Error Connecting:", errc)
+    except Timeout as errt:
+        print("Timeout Error:", errt)
+    except RequestException as err:
+        print("Something Else:", err)
+
+    return ''
+
+
 response = request_data(100, start, end)
-sorted_site = sorted(response.json()['results'], key=lambda i: i['pub_date'])
-site_headlines = sorted([x['headline'] for x in sorted_site])
 
-submitted = set(sheet_headlines) & set(site_headlines)
-missing = list(sorted(set([x.strip() for x in site_headlines]) - set([x.strip() for x in sheet_headlines])))
-missing_full = sorted([x for x in sorted_site if x['headline'] in missing], key=lambda i: i['headline'])
+if response.status_code == 200:
+    print('Request is good.')
+    sorted_site = sorted(response.json()['results'], key=lambda i: i['pub_date'])
+    site_headlines = sorted([x['headline'] for x in sorted_site])
+
+    submitted = set(sheet_headlines) & set(site_headlines)
+    missing = list(sorted(set([x.strip() for x in site_headlines]) - set([x.strip() for x in sheet_headlines])))
+    missing_full = sorted([x for x in sorted_site if x['headline'] in missing], key=lambda i: i['headline'])
+else:
+    print(f'Request is not good: {response.status_code}.')
```

Comment:
Looks like a syntax error here. Maybe `print('Request is not good: {}.'.format(response.status_code))`?